db/system_keyspace.cc

```diff
@@ -876,6 +892,7 @@ schema_ptr system_keyspace::v3::views_builds_in_progress() {
        );
        builder.set_gc_grace_seconds(0);
        builder.with_version(generate_schema_version(builder.uuid()));
+       builder.set_wait_for_sync_to_commitlog(true);
```

Comment:
@psarna comment

Reply:
Actually, it seems that it doesn't prevent anything, since `views_builds_in_progress` is also virtual.

Reply:
The flag prevents power less or crash from losing an acknowledge write (commitlog periodic mode vs batch mode)